example/android/app/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,7 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
     <application
-        android:name="io.flutter.app.FlutterApplication"
+        android:name="${applicationName}"
```

Comment:
This breaks local builds of example project, fails with
```
/Volumes/Data/Projects/github.com/flutter-mapbox-gl/maps/example/android/app/src/main/AndroidManifest.xml:8:9-42 Error:
	Attribute application@name at AndroidManifest.xml:8:9-42 requires a placeholder substitution but no value for <applicationName> is provided.
/Volumes/Data/Projects/github.com/flutter-mapbox-gl/maps/example/android/app/src/main/AndroidManifest.xml Error:
	Validation failed, exiting
```

Reply:
whats your flutter version? It build fine for me

Reply:
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.5.3, on macOS 12.2 21D49 darwin-x64, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[!] Android Studio
    ✗ Android Studio not found at /Volumes/Data/Applications/Android Studio 4.0.1.app/Contents
[✓] VS Code (version 1.63.2)
[✓] VS Code (version 1.49.1)
[✓] Connected device (3 available)

```

Reply:
can you try 2.8?

Reply:
Worked on 2.8.1, thanks.